tests/test_account_chart.py

```diff
@@ -0,0 +1,99 @@
+"""
+Unit tests for account charts accessor and mutator methods.
+"""
+
+import pytest
+from cashctrl_ledger import CashCtrlLedger
+
+# Ensure there is no '2222' account on the remote system
+def test_delete_account_non_existent():
+    cashctrl_ledger = CashCtrlLedger()
+    cashctrl_ledger.delete_account(7777, allow_missing=True)
+    assert 7777 not in cashctrl_ledger.account_chart().index
+
+# Test adding a valid account
+def test_add_account():
+    cashctrl_ledger = CashCtrlLedger()
+    initial_accounts = cashctrl_ledger.account_chart().reset_index()
+    new_account = {
+        'account': 7777,
+        'currency': 'CHF',
+        'text': 'test create account',
+        'vat_code': 'MwSt. 2.6%',
+        'group': '/Anlagevermögen'
+    }
+    cashctrl_ledger.add_account(**new_account)
+    updated_accounts= cashctrl_ledger.account_chart().reset_index()
+    created_accounts = updated_accounts[
+        ~updated_accounts.apply(tuple, 1).isin(initial_accounts.apply(tuple, 1))]
```

Comment:
Now that I am more familiar with performing set operations on DataFrames, I prefer the following logic:
```
outer_join = pd.merge(initial_accounts, updated_accounts, how='outer', indicator=True)
created_accounts = outer_join[outer_join['_merge'] == "right_only"].drop('_merge', axis = 1)
```
Difference:
- The proposed code does not depend on identical columns nor identical column sequence.
- I find the proposed version easier to read.

Please replace similar occurrences in this unit and in the unit test for VAT.

Reply:
Thanks for implementing in `tests/test_account_chart.py`
Please also replace similar occurrences in `tests/test_vat_code.py`